### PR TITLE
[CHORE] Fix the build

### PR DESCRIPTION
**Pull request by cyril-sf**
The build is failing because `ember-cli-page-object` 1.15.0 ( https://github.com/san650/ember-cli-page-object/releases/tag/v1.15.0 ) `includes support for RFC232 and RFC268 style tests. This mode is automatically enabled when you use new @ember/test-helpers's setupApplicationTest/setupRenderingTest.`

#13 fails because a dependency is not properly pinned and any new build would currently fail.

```
not ok 10 Chrome 76.0 - Integration | Component | color picker cell: Color cell renders with background color
    ---
        actual: >
            null
        stack: >
            Error: Must setup rendering context before attempting to interact with elements.
                at Object.getRootElement (http://localhost:7357/assets/test-support.js:8373:13)
                at getRootElement (http://localhost:7357/assets/test-support.js:11003:49)
                at ExecutionContext.getElements (http://localhost:7357/assets/test-support.js:12003:81)
                at ExecutionContext.findWithAssert (http://localhost:7357/assets/test-support.js:11988:25)
                at findElementWithAssert (http://localhost:7357/assets/test-support.js:12725:72)
                at Object.get (http://localhost:7357/assets/test-support.js:13253:58)
                at Object.style (http://localhost:7357/assets/test-support.js:1730:31)
                at Object.isColor (http://localhost:7357/assets/test-support.js:7233:19)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:182:20)
                at runTest (http://localhost:7357/assets/test-support.js:3723:30)
```

I'm pinning the dependency instead of migrating the tests ( https://travis-ci.org/Addepar/addepar-forms/builds/582967347 ) as the tests fail on Ember 1.13 ( https://travis-ci.org/Addepar/addepar-forms/jobs/582967349 )

```
not ok 10 Chrome 76.0 - adde-color-picker-cell: Color cell renders with background color
    ---
        actual: >
            null
        stack: >
            TypeError: owner.lookup is not a function
                at http://localhost:7357/assets/test-support.js:10017:32
                at tryCatch (http://localhost:7357/assets/vendor.js:67368:14)
                at invokeCallback (http://localhost:7357/assets/vendor.js:67383:15)
                at publish (http://localhost:7357/assets/vendor.js:67351:9)
                at http://localhost:7357/assets/vendor.js:43725:7
                at invoke (http://localhost:7357/assets/vendor.js:11245:16)
                at Queue.flush (http://localhost:7357/assets/vendor.js:11309:11)
                at DeferredActionQueues.flush (http://localhost:7357/assets/vendor.js:11110:17)
                at Backburner.end (http://localhost:7357/assets/vendor.js:10399:25)
                at http://localhost:7357/assets/vendor.js:10990:18
        message: >
            Promise rejected before "Color cell renders with background color": owner.lookup is not a function
```

Also update to node 8 and add Travis badge to README.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
   "name": "@addepar/forms",
-  "version": "0.0.0",
+  "version": "0.0.1",
```
Comment (mixonic) [inline package.json]: Why?

**Comment by cyril-sf** [comment]
@mixonic #13 fails because a dependency is not properly pinned and any new build would currently fail.

```
not ok 10 Chrome 76.0 - Integration | Component | color picker cell: Color cell renders with background color
    ---
        actual: >
            null
        stack: >
            Error: Must setup rendering context before attempting to interact with elements.
                at Object.getRootElement (http://localhost:7357/assets/test-support.js:8373:13)
                at getRootElement (http://localhost:7357/assets/test-support.js:11003:49)
                at ExecutionContext.getElements (http://localhost:7357/assets/test-support.js:12003:81)
                at ExecutionContext.findWithAssert (http://localhost:7357/assets/test-support.js:11988:25)
                at findElementWithAssert (http://localhost:7357/assets/test-support.js:12725:72)
                at Object.get (http://localhost:7357/assets/test-support.js:13253:58)
                at Object.style (http://localhost:7357/assets/test-support.js:1730:31)
                at Object.isColor (http://localhost:7357/assets/test-support.js:7233:19)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:182:20)
                at runTest (http://localhost:7357/assets/test-support.js:3723:30)
```

I'm still trying to decide if I simply pin `ember-cli-page-object`  or if I can update the tests to be compatible from Ember 1.13 to the latest Ember.

File: package.json (hunk 1 above)
Comment (cyril-sf) [reply package.json]: I saw that a 0.0.1 release was already existing.

**Comment by mixonic** [comment]
@cyril-sf so is the suggestion that this is good to go with a failing CI suite, or do you plan to address the failure first?

**Comment by cyril-sf** [comment]
@mixonic it still needs to get green.

**Pull request merged**